std/typecons.d

```diff
@@ -5495,11 +5495,11 @@ if (!is(T == class) && !(is(T == interface)))
             import core.exception : onOutOfMemoryError;
             import std.conv : emplace;
 
-            _store = cast(Impl*) pureMalloc(Impl.sizeof);
+            _store = (() @trusted => cast(Impl*) pureMalloc(Impl.sizeof))();
```

Comment:
`@trusted` for the cast from void* to Impl*. Could only be unsafe if _store were dereferenced before being initialized, which cannot happen since we next call emplace.

Reply:
These comments should be put in the source code, s.t. they are easy to catch by future readers ;-)